plugins/woocommerce-admin/client/products/hooks/use-product-attributes.ts

```diff
@@ -78,12 +78,21 @@ export function useProductAttributes( {
 		};
 	};
 
+	const getAugmentedAttributes = ( atts: ProductAttribute[] ) => {
+		return atts.map( ( attribute, index ) => ( {
+			...attribute,
+			variation: isVariationAttributes,
```

Comment:
Is `variation` a boolean? Is `isVariationAttributes` a boolean? The naming is making this simple line more difficult for my brain than it should be!

Reply:
Ya, this is unfortunate naming.  Both are booleans, but `variation` is pre-existing in the [attribute schema](https://woocommerce.github.io/code-reference/classes/WC-Product-Attribute.html), so we can't do much about that right now.